src/ZoomLoop.Api/Features/VehicleValuation/GetVehicleValuation.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Quinntyne Brown. All Rights Reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using MediatR;
+
+namespace ZoomLoop.Api.Features.VehicleValuation;
+
+public record GetVehicleValuationRequest(
+    string Vin,
+    int Year,
+    string Make,
+    string Model,
+    string PostalCode,
+    int Kilometers,
+    int Accidents,
+    string InteriorCondition,
+    string ExteriorCondition) : IRequest<GetVehicleValuationResponse>;
+
+public class GetVehicleValuationResponse
```

Comment:
Adhere to Single class per file rule. Any file with multiple classes shall be split to two files